### PR TITLE
Add hack/dep.py to check dep status against k8s Godeps.json

### PR DESCRIPTION
**Pull request by justinsb**

### CODE REVIEW

File: hack/dep.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+# It generates bash commands where changes are needed
+# We can probably also use it for deps when the time comes!
+
+import json
+import sys
+import subprocess
+from pprint import pprint
+from os.path import expanduser, join
+
+kops_dir = expanduser('~/k8s/src/k8s.io/kops')
```
Comment (geojaz) [inline hack/dep.py]: Please base these paths on the go envs, not your directory structure :)

**Comment by justinsb** [comment]
Updated - want to take another look @geojaz ?

**Comment by justinsb** [comment]
And, the eventual plan is to migrate to dep, once it is ready.  Right now it isn't ready - it crashes when we try to use it on something of k8s or kops's size.

**Comment by justinsb** [comment]
protoc issue

/retest

**Comment by justinsb** [comment]
We merged a similar fix for this in #4382.  Closing.

**Pull request closed without merging**